cmd/zrobot

```diff
@@ -23,4 +23,12 @@ entry_point.add_command(blueprint)
 entry_point.add_command(godtoken)
 
 if __name__ == "__main__":
+    from jumpscale import j
```

Comment:
you cannot import jumpscale here because we monkey patch later on: in https://github.com/threefoldtech/0-robot/blob/development/zerorobot/cli/server.py#L2
So if you import jumpscale there, the monkey patching will happen too late